src/main/kotlin/data/UserPreferences.kt

```diff
@@ -0,0 +1,13 @@
+package data
+
+data class UserPreferences(
+        var id: Int = 0,
+        var preferredSourceLanguage: Language,
+        var preferredTargetLanguage: Language,
+        var dayNightMode: Enum<DayNight> = DayNight.DAY,
+        var uiLanguagePreferences: String = "en"
```

Comment:
what?

Reply:
We thought that for users that are able to read we could store their preferred language to see the UI in, however we could just go off the system Locale